<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutPutTabWidgetClass</class>
 <widget class="QFrame" name="OutPutTabWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>585</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageVideoWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layoutOutputControl">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="DeviceVideoWidget" name="videoWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="ConfigPanoramaWidget" name="panoramaConfigurationWidget"/>
     <widget class="AudioProcessorsWidget" name="audioProcessorsConfigurationWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceVideoWidget</class>
   <extends>QWidget</extends>
   <header>libvideostitch-gui/widgets/devicevideowidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigPanoramaWidget</class>
   <extends>QWidget</extends>
   <header>src/configurations/configpanorama.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioProcessorsWidget</class>
   <extends>QWidget</extends>
   <header>src/widgets/audioprocessors/audioprocessorswidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
